__tests__/app.test.js

```diff
@@ -80,6 +80,25 @@ describe("GET /api/articles", () => {
                 });
             });
     });
+    test("200 returns array of articles filtered by topic query", () => {
+        return request(app)
+            .get("/api/articles?topic=mitch")
+            .expect(200)
+            .then(({ body }) => {
+                const { articles } = body;
+                articles.forEach((article) => {
+                    expect(article.topic).toBe("mitch");
+                });
+            });
+    });
+    test("400 returns invalid topic query", () => {
+        return request(app)
+        .get("/api/articles?topic=invalid_topic")
+        .expect(400)
+        .then(( { body }) => {
+            expect(body.msg).toBe("invalid query");
+        })
+    })
```

Comment:
Please add one more 200 test for a topic which exists but has no articles. We would like an empty array to be returned in this case :) 

I believe topic=paper should do this 👍